modules/example/src/test/scala-2/example/AnalysisTestSpecs2.scala

```diff
@@ -22,6 +22,6 @@ class AnalysisTestSpecs2 extends Specification with IOChecker {
   // check(AnalysisTest.arrayTest2)
   check(AnalysisTest.pointTest)
   // check(AnalysisTest.pointTest2)
-  // check(AnalysisTest.update("foo", 42))
+  checkOutput(AnalysisTest.update("foo", "bkah"))
```

Comment:
I wanted to test on [IOCheckerCheck](https://github.com/tpolecat/doobie/blob/main/modules/specs2/src/test/scala/doobie/specs2/CheckerTests.scala#L30), but this one seemed to be set up to test only simple queries, so I am testing on the example project.

Reply:
It's ok :) I think at some point we need to realign all the tests (scalatest/munit/weaver etc etc) so they test all the scenarios